gulpfile.js

```diff
@@ -106,39 +112,80 @@ gulp.task('translations-feelup', ['prepare-translations'], function () {
                     item.msgstr = [''];
                 }
                 if (0 == item.msgstr[0].length) {
-//                    item.msgid[0] = item.msgid;
+                    // item.msgid[0] = item.msgid;
                     item.msgstr[0] = item.msgid;
                 }
                 return item;
             }
         }))
         .pipe(gulp.dest(languagesFolder));
-});
+}
 
 // Cleanup temporary translation folders.
-gulp.task('cleanup', ['prepare-translations'], function () {
-    var folders = getFolders(languagesFolder);
-
+function cleanup() {
+    prepare_translations();
     return folders.map(function (folder) {
         return gulp.src(path.join(languagesFolder, folder), {read: false})
             .pipe(clean());
     });
-});
+}
 
 // Compile *.po to *.mo binaries for usage.
-gulp.task('compile-translations', ['translations-feelup'], function () {
+function compile_translations() {
+    translations_feelup();
     // Compile POs to MOs.
     return gulp.src(languagesFolder + '*.po')
         .pipe(gettext())
         .pipe(gulp.dest(languagesFolder))
-});
-
-gulp.task('default', [], function () {
-    gulp.run('prepare-source');
-    gulp.run('update-transifex');
-    gulp.run('download-translations');
-    gulp.run('prepare-translations');
-    gulp.run('translations-feelup');
-    gulp.run('cleanup');
-    gulp.run('compile-translations');
-});
\ No newline at end of file
+}
+
+function style() {
+	let plugins = [
+		cssnano()
+	];
+
+	// location of style
+	return gulp.src( './assets/scss/**/*.scss' )
+		// Compile file
+		.pipe( sass() )
+		// Use postcss
+		.pipe( postcss( plugins ) )
+		// Push build
+		.pipe( gulp.dest( './assets/css/admin' ) )
```

Comment:
@bahiirwa this will create files inside `assets/css/admin/`. So some files will be nested as `assets/css/admin/admin`, like you can see in the diff of the PR. Please target `assets/css`.

Reply:
Thanks for spotting this. I had two rules for compiling the admin css separately from the front end. Seems, I forgot to refactor this when I joined the two rules.
Fixed it.